torchrl/objectives/ppo.py

```diff
@@ -506,6 +527,16 @@ def loss_critic(self, tensordict: TensorDictBase) -> torch.Tensor:
                 f"can be used for the value loss."
             )
 
+        if self.clip_value:
+            try:
+                old_state_value = tensordict.get(self.tensor_keys.value).clone()
```

Comment:
why do we clone here?

Reply:
Because the `self.tensor_keys.value` prediction will be recomputed once we pass tensordict through the value network and we will lose this tensor.